core/corepb/v1/consensus.proto

```diff
@@ -13,14 +13,28 @@ message QBFTMsg {
   core.corepb.v1.Duty            duty                  = 2;
   int64                          peer_idx              = 3;
   int64                          round                 = 4;
-  reserved                                               5; // Deprecated value
+  core.corepb.v1.UnsignedDataSet value_legacy          = 5 [deprecated = true]; // Prefer value.
```

Comment:
these fields need to be here, since they are included when calculating signature hashes. Removing them results in server and client calculating different hashes.